var/spack/repos/builtin/packages/genfit/package.py

```diff
@@ -7,14 +7,21 @@
 
 
 class Genfit(CMakePackage):
-    """GenFit is a tracking framework in particle and nuclear physics."""
+    """GenFit is an experiment-independent framework for track reconstruction in
+        particle and nuclear physics"""
 
     homepage = "https://github.com/GenFit/GenFit"
-    url      = "https://github.com/GenFit/GenFit/archive/master.zip"
+    url      = "https://github.com/GenFit/GenFit/archive/02-00-00.tar.gz"
     git      = "https://github.com/GenFit/GenFit.git"
 
     maintainers = ['mirguest']
 
     version('master', branch='master')
+    version('02-00-00', sha256='0bfd5dd152ad0573daa4153a731945824e0ce266f844988b6a8bebafb7f2dacc')
+    # Untagged version from 2017-06-23 known to work with root@6.16.00
+    version('b496504a', sha256='e1582b35782118ade08498adc03f3fda01979ff8bed61e0520edae46d7bfe477')
 
     depends_on('root')
+    depends_on('eigen')
+
+    conflicts('root@6.18.00:', when='@b496504a')
```

Comment:
@alalazo shouldn't this be:
```suggestion
    conflicts('^root@6.18.00:', when='@b496504a')
```

Reply:
I think we shouldn't be using a conflict in this case, but a normal depends on.